website/docs/foundations/accessibility/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Accessibility Foundations
```

Comment:
```suggestion
title: Accessibility
```
_Foundations_ feels redundant given we're in the foundations section already

Reply:
I wanted to make it "Foundations: Accessibility" but I can't put colons in frontmatter. Any ideas? Goal is to have a unique page title. Unless we are merging page titles?

Reply:
unique throughout the entire site?

Reply:
Yes; ideally we'll have something like `Page Name | Category Name | Site Name` (category/section name if it exists)

Reply:
Does "title" here refer to what is displayed in the cover or the HTML `<title>` tag?

Reply:
This is the title that appears in the page "cover". 
The page title (the one in the `<head>` comes from this function here:
https://github.com/hashicorp/design-system/blob/main/website/app/services/head-data.js#L9-L15
so we can customize it as we want (more or less, it depends on the data available at that level in the application).

Reply:
👍  i have a task in the backlog related to the titles not working correctly in certain cases. Can make this update at the same time.